src/Tizen.NUI/src/public/NUIConstants.cs

```diff
@@ -746,7 +746,13 @@ public enum WindowType
         /// <summary>
         /// Used for simple dialog windows.
         /// </summary>
-        Dialog
+        Dialog,
+        /// <summary>
+        /// Suppors to make partial size window easliy.
```

Comment:
Suppors -> Supports ?

Reply:
Thank you for detail review.
BTW, This PR is not needed any more in API 8.
I will prepare new PR for the latest version.
It will be closed.